targets/posix/nanoCLR/nanoCLR_native_posix.cpp

```
@@ -0,0 +1,205 @@
+
+void nanoCLR_SetWireProtocolReceiveCallback(WireReceiveCallback receiveCallback)
+{
+    WireProtocolReceiveCallback = receiveCallback;
+}
+
+void nanoCLR_SetWireProtocolTransmitCallback(WireTransmitCallback transmitCallback)
+{
+    WireProtocolTransmitCallback = transmitCallback;
+}
+
+void nanoCLR_WireProtocolProcess()
```

Cppcheck (2.20.0)

[style] The function 'nanoCLR_WireProtocolProcess' is never used.

(unusedFunction)